galley/pkg/server/server.go

```diff
@@ -108,6 +110,9 @@ func newServer(a *Args, p patchTable) (*Server, error) {
 		if err != nil {
 			return nil, err
 		}
+		if err := p.createResourceTypes(k); err != nil {
+			return nil, err
```

Comment:
How do we deal with uninstallation? It looks like this will only try to create once per startup but will not remove them. This will get broken fairly quickly I think.

Reply:
I don't think we need to optimize for the uninstall case given the risk. Manually cleaning up the CRD's isn't too much work if we label the CRD's properly, e.g.

```bash
kubectl delete crd -l app=mixer
kubectl delete crd -l app=istio-pilot
kubectl delete crd -l app=istio-mixer
```